bootstrap/12-pm.sh

```diff
@@ -143,25 +143,26 @@ if ${XFCE4}; then
     configure_keyboard_configuration
 fi
 
-if [ ! -z ${INCLUDES} ]; then
-    packages_list=`echo ${INCLUDES} | sed -r 's/,/ /g'`
+if [[ -n ${INCLUDES} ]]; then
+    packages_list=$(echo "${INCLUDES}" | sed -r 's/,/ /g')
 
-    run_scripts ${SOURCE_DIR}/pre-install
+    run_scripts "${SOURCE_DIR}"/pre-install
 
     info "installing ${packages_list}"
+    # shellcheck disable=SC2086
```

Comment:
What's wrong with this?

Reply:
`${packages_list}` mustn't be quoted. For example, `${packages_list}` stores `nginx mariadb-server-10.3`. If you invoke `install_packages "${packages_list}"`, the package manager will try to find and install the package named `"nginx mariadb-server-10.3"` but not 1) `nginx` and 2) `mariadb-server-10.3`. However, shellcheck doesn't like when variables are unquoted=)

Reply:
I added a comment to the line.